pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/PreparedStatement.java

```diff
@@ -72,6 +72,15 @@ public ResultSetGroup execute() {
     return _connection.execute(new Request(_queryFormat, fillStatementWithParameters()));
   }
 
+  /**
+   * Executes this prepared statement and returns the BrokerResponse.
+   *
+   * @return The BrokerResponse received.
+   */
+  public BrokerResponse executeRequest() {
```

Comment:
@kishoreg @npawar I'm not super convinced with this name but can't think of a better name than this. Feel free to suggest something else.

Reply:
Don't rename any methods. Since it won't be backward compatible then.

Reply:
This isn't a rename. It's a new method addition.

Reply:
This isn't applicable anymore as per the latest change.